File: doc/src/scripts/Interacting.java
import krpc.client.Connection;
import krpc.client.RPCException;
import krpc.client.services.SpaceCenter;
import krpc.client.services.SpaceCenter.Vessel;

import java.io.IOException;

public class Interacting {
    public static void main(String[] args) throws IOException, RPCException {
        Connection connection = Connection.newInstance("Vessel Name");
        SpaceCenter spaceCenter = SpaceCenter.newInstance(connection);
        Vessel vessel = spaceCenter.getActiveVessel();
        System.out.println(vessel.getName());
    }
}

File: doc/src/scripts/Interacting.cs
using KRPC.Client;
using KRPC.Client.Services.SpaceCenter;

class Program
{
    public static void Main ()
    {
        using (var connection = new Connection (name : "Vessel Name")) {
            var spaceCenter = connection.SpaceCenter ();
            var vessel = spaceCenter.ActiveVessel;
            System.Console.WriteLine (vessel.Name);
        }
    }
}
